specification/eventhub/resource-manager/Microsoft.EventHub/preview/2022-01-01-preview/ApplicationGroups.json

```diff
@@ -0,0 +1,322 @@
+          {
+            "$ref": "../../../common/v1/definitions.json#/parameters/NamespaceNameParameter"
+          },
+          {
+            "$ref": "../../../common/v2/definitions.json#/parameters/ApplicationGroupNameParameter"
+          },
+          {
+            "$ref": "../../../common/v1/definitions.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../common/v1/definitions.json#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Namespace ApplicationGroup returned successfully.",
+            "schema": {
+              "$ref": "#/definitions/ApplicationGroup"
+            }
+          },
+          "default": {
+            "description": "Eventhub error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../common/v2/definitions.json#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "ApplicationGroupListResult": {
+      "type": "object",
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ApplicationGroup"
+          },
+          "description": "Result of the List Application Groups operation."
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "Link to the next set of results. Not empty if Value contains an incomplete list of Authorization Rules"
+        }
+      },
+      "description": "The response from the List Application Groups operation."
+    },
+    "ApplicationGroup": {
+      "type": "object",
+      "properties": {
+        "properties": {
+          "type": "object",
+          "x-ms-client-flatten": true,
+          "properties": {
+            "updatedAt": {
+              "readOnly": true,
+              "format": "date-time",
+              "type": "string",
+              "description": "Exact time the Application Group was updated"
+            },
+            "createdAt": {
+              "readOnly": true,
+              "format": "date-time",
+              "type": "string",
+              "description": "Exact time the Application Group was created."
+            },
+            "isEnabled": {
+              "type": "boolean",
+              "description": "Determines if Application Group is allowed to create connection with namespace or not. Once the isEnabled is set to false, all the existing connections of application group gets dropped and no new connections will be allowed"
+            },
+            "clientAppGroupIdentifier": {
+              "type": "string",
+              "description": "The Unique identifier for application group.Supports SAS(SASKeyName=KeyName) or AAD(AADAppID=Guid1)"
+            },
+            "policies": {
+              "type": "array",
+              "items": {
+                "$ref": "#/definitions/ThrottlingPolicy"
```

Comment:
>ThrottlingPolicy [](http://example.com/codeflow?start=39&length=16)

Can multiple policies be set for each MetricId? How do u resolve such conflicts?

Reply:
If you can only have a single value for each MetricId, consider making Policies a child resource

Reply:
No, we cannot set multiple policies for each metricId.
I did not understand the child resource part, as in /ApplicationGroups/Policies ? Then it would conflict with our design and how we want the end user to update the applicationGroup and not the policies directly.